### DISCUSSION TITLE
How to transfer USDT with Contract using walletconnect

### DISCUSSION THREAD

**Discussion by liguohui-lgh** [WalletConnect/walletconnect-monorepo#1419] [Developer Support] [answered]
Hi, I Have connected to MetaMask by WalletConnect on my mobile phone. In addition, I can trans native currency by using `connector.sendTransaction` which rpc method is `eth_sendTransaction`.  However, I don't know how to transfer ERC-20 Token by using WalletConnect. Thanks!!!
Tip: My app is running on mobile phone's Chrome browser.

**Comment by hackyguru** [comment, marked as answer]
Hi @liguohui-lgh , you can send ERC-20 tokens of your choice by mentioning the `chainId` and `contract address` pertaining to the token in the tx object :)

**Reply by lbudakov** [reply]
An example will be much appreciated

**Reply by nikitaNotFound** [reply]
@hackyguru @liguohui-lgh @lbudakov hey guys! Do you have any results with finding examples? I was able to call smart contract for USDT transfer, but UI that wallets show (TrustWallet, BitKeep) is not user friendly at all.
